samples: ipc: static_vrings: Exclude unsupported platforms

doki-nordic opened the pull request:
List of allowed platforms for this sample was changed in PR #85997. This enabled more platforms including those that are not needed. This commit excludes nrf5340/cpuapp_ns and nrf5340/cpunet platforms since they are not supported by this sample.

katgiadla commented on samples/subsys/ipc/ipc_service/static_vrings/sample.yaml:
```diff
@@ -3,6 +3,9 @@ sample:
   sample.ipc.static_vrings:
     filter: dt_compat_enabled("zephyr,ipc-openamp-static-vrings")
+    platform_exclude:
+      - nrf5340dk/nrf5340/cpuapp_ns
```
Please rename the platform:
```suggestion
      - nrf5340dk/nrf5340/cpuapp/ns
```

doki-nordic replied:
Done

dkalowsk commented:
@doki-nordic this needs a rebase for 4.2 inclusion

henrikbrixandersen commented:
Please rebase on `main` and resolve any merge conflicts.

doki-nordic committed:
samples: ipc: static_vrings: Exclude unsupported platforms

List of allowed platforms for this sample was
changed in PR #85997. This enabled more
platforms including those that are not needed.
This commit excludes nrf5340/cpuapp_ns and
nrf5340/cpunet platforms since they are not
supported by this sample.

Signed-off-by: Dominik Kilian <Dominik.Kilian@nordicsemi.no>
